src/i18n/index.js

```diff
@@ -68,7 +70,8 @@ const numberLocales = {
   // [LANGUAGES.CHINESE_TRADITIONAL]: customNumberFmt,
   [LANGUAGES.KOREAN]: defaultNumberFmt,
   [LANGUAGES.JAPANESE]: defaultNumberFmt,
-  [LANGUAGES.RUSSIAN]: russianNumberFmt,
+  [LANGUAGES.RUSSIAN]: decimalCommaNumberFmt,
+  [LANGUAGES.SPANISH]: decimalCommaNumberFmt,
```

Comment:
@v-almonacid this seems off

`groupSeparator: '  ',` it would look like: 1 000,20 and it should look like 1.000,20

Reply:
Thanks for the heads up! You are right, I used the same thousands separator we were using for Russian, but it's not really common in Spanish (except in Peru it seems :P).